integration-tests/actor-tests/uniswap.test.ts

```diff
@@ -50,10 +50,10 @@ actor('Uniswap swapper', () => {
   setupRun(async () => {
     const wallet = Wallet.createRandom().connect(env.l2Provider)
 
-    await env.l2Wallet.sendTransaction({
+    await (await env.l2Wallet.sendTransaction({
```

Comment:
I think nesting awaits is generally an antipattern, makes the whole promise of easy to read control flow difficult

Reply:
Agreed! Fixed.